index.js

```diff
@@ -1 +1,717 @@
+  res.redirect('/login');
+});
+
+/* ================================================= CHECK FOR COOKIES */
+app.all('*', checkIfLoggedIn, (req, res, next) => {
+  console.log('In check route');
+
+  // if there is a userName cookie it means there is a login cookie
+  if (req.isUserLoggedIn) {
+    return next();
+  }
+
+  // if don't have login cookie
+  console.log('Unverified user! Proceed to login page');
+  // redirect to login
+  res.redirect('/login');
+});
+
+/* ======================================================= OTHER PAGES */
+// ------------------------------ HOMEPAGE
+app.get('/', getUserNameAndIdCookie, (req, res) => {
+  // get cookie data
+  const { userName } = req;
+  const { userId } = req;
+
+  // get data
+  const sqlQuery = 'SELECT id, date_of_sighting, flock_size FROM notes';
+  pool.query(sqlQuery, (error, result) => {
+    if (error) {
+      console.log('Error executing homepage query', error.stack);
+      res.status(503).send(result.rows);
+      return;
+    }
+
+    const dataArr = { arr: result.rows };
+
+    // the date is in ISO 8601 format. we need to use moment to convert
+    // it to a readable format for the form
+    let dateOfSightingArr = [];
+    dataArr.arr.forEach((e, i) => {
+      const refomatDateOfSighting = convertDate(e.date_of_sighting, 'ISO8601', 'YYYY-MM-DD');
+      dateOfSightingArr.push(refomatDateOfSighting);
+    });
+    dateOfSightingArr = { sightingArr: dateOfSightingArr };
+
+    // render
+    res.render('index', {
+      dataArr, dateOfSightingArr, userName, userId,
+    });
+  });
+});
+
+// ------------------------------- HOMEPAGE WITH SORTING
+app.get('/:sort', getUserNameAndIdCookie, (req, res, next) => {
+  // get cookie data
+  const { userName } = req;
+  const { userId } = req;
+  // sort database function
+  const sortDataBase = (typeOfSortSql) => {
+    const sqlQuery = `SELECT * FROM notes ORDER BY date_of_sighting ${typeOfSortSql}`;
+    pool.query(sqlQuery, (error, result) => {
+      if (error) {
+        console.log('Error executing homepage query', error.stack);
+        res.status(503).send(result.rows);
+        return;
+      }
+      const dataArr = { arr: result.rows };
+      // the date is in ISO 8601 format. we need to use moment to convert
+      // it to a readable format for the form
+      let dateOfSightingArr = [];
+      dataArr.arr.forEach((e, i) => {
+        const refomatDateOfSighting = convertDate(e.date_of_sighting, 'ISO8601', 'YYYY-MM-DD');
+        dateOfSightingArr.push(refomatDateOfSighting);
+      });
+      dateOfSightingArr = { sightingArr: dateOfSightingArr };
+      // render
+      res.render('index', {
+        dataArr, dateOfSightingArr, userName, userId,
+      });
+    });
+  };
+
+  // get query param
+  const { sort } = req.params;
+  switch (sort) {
+    case 'sort=asc':
+    // sort data base
+      sortDataBase('ASC');
+      break;
+    case 'sort=desc':
+    // code block
+      sortDataBase('DESC');
+      break;
```

Comment:
since there are only 2 cases, is there a reason why you used switch above if, else?